src/renderer/component/subscribeButton/view.jsx

```diff
@@ -17,23 +17,31 @@ type Props = {
   doChannelSubscribe: ({ channelName: string, uri: string }) => void,
   doChannelUnsubscribe: SubscribtionArgs => void,
   doNotify: ({ id: string }) => void,
+  firstRunCompleted: boolean,
 };
 
 export default (props: Props) => {
   const {
-    channelName,
```

Comment:
Before we were passing the uri and the channel name in, but we can just get the channel name from the uri